### PR TITLE
zwave: add Fibaro Dimmer 2 (FGD-212) to DB

### PR DESCRIPTION
**Pull request by mattnl**

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/fibaro/fgd212.xml [hunk 1]
```diff
@@ -0,0 +1,871 @@
...
+			<SetToController>false</SetToController>
+		</Group>
+		<Group>
+			<Index>4</Index>
+			<Maximum>10</Maximum>
+			<Label lang="en">Sends BASIC SET command class frame according to the state of the device. The Dimmer in 2nd to 5th group allows for controlling 5 regular and 5 multichannel devices per an association group, out of which 1 field is reserved for the Z-Wave network main controller.</Label>
+			<SetToController>false</SetToController>
+		</Group>
+		<Group>
+			<Index>5</Index>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/fibaro/fgd212.xml]: Should have maximum 10

**Comment by cdjackson** [comment]
Thanks - just one small issue. If you can correct this I'll merge both requests...

**Comment by mattnl** [comment]
Thanks Chris, done.

**Comment by cdjackson** [comment]
Thanks.

**Pull request merged**